books/api/credits_notes.py

```diff
@@ -173,7 +173,7 @@ def update_billing_address(self,creditnote_id:str, data:dict):
                 "fax": "+1-925-924-9600"
             }
         """
-        return self.base.api._standard_call(f'{self.module}/{creditnote_id}/address/billing','put')
+        return self.base.api._standard_call(f'{self.module}/{creditnote_id}/address/billing','put', data=data)
```

Comment:
data.json? or it is not necessary?

Reply:
Im not sure, maybe like data is dict is not neccesary

Reply:
maybe just need convert to str